extractors/bilibili/bilibili.go

```diff
@@ -512,7 +512,7 @@ func getSubTitleCaptionPart(aid int, cid int) *extractors.CaptionPart {
 	}
 	return &extractors.CaptionPart{
 		Part: extractors.Part{
-			URL: stu.Data.SubtitleInfo.SubtitleList[0].SubtitleUrl,
+			URL: fmt.Sprintf("https:%s", stu.Data.SubtitleInfo.SubtitleList[0].SubtitleUrl),
```

Comment:
Basically LGTM, I'm a little worried, is the URL returned here something like `//xx.com`?

Reply:
> Basically LGTM, I'm a little worried, is the URL returned here something like `//xx.com` not `https://xx.com`?

Yes, I found a video with subtitles and tried it out.